tools/ci/update-vulnerable-dependencies.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+command -v osv-scanner >/dev/null 2>&1 || { echo >&2 "osv-scanner not installed!"; exit 1; }
```

Comment:
Can we have an issue to have an installation script add this to `~/.kuma-dev/bin/` (dev tools dir)?

Reply:
This would force us to run `make dev/tools` and install everything in this action but that's ok I guess :shrug:

Reply:
Yeah, that's why I wanted to have an issue to discuss this with the team. Actually we don't have to run `make dev/tools` I think we could just run individual install scripts.

Reply:
I tried to add a ["install only these packages" feature](https://github.com/kumahq/kuma/pull/5420#issuecomment-1341101584) once but I think @lahabana was right in the end, maybe just add a cache step here and install everything. I'll add `jq` and `osv-scanner` in a new PR.

Reply:
I tried to add a ["install only these packages" feature](https://github.com/kumahq/kuma/pull/5420#issuecomment-1341101584) once but I think @lahabana was right in the end, maybe just add a cache step here and install everything. I'll add `jq` and `osv-scanner` in a new PR.

Reply:
OK let's discuss in https://github.com/kumahq/kuma/issues/6111